src/app/shared/palette/collection/write-collection-field.component.ts

```diff
@@ -72,4 +74,27 @@ export class WriteCollectionFieldComponent extends AbstractFieldWriteComponent i
     return index ? `${this.caseField.label} ${displayIndex}` : this.caseField.label;
   }
 
+  openModal(i: number) {
```

Comment:
can we give i a more meaningful name?

Reply:
i is basically the index of the items within the collection type field.